### ISSUE TITLE
Video player

### ISSUE THREAD

**Issue by Code-Slave** [meeb/tubesync#124]
Aince you have all this painstakingly pulled meta data etc whats thought about a frontend to actually play the videos?

a aimple channel list on left nav and thumbnails on right that open a player. ?

**Comment by meeb** [comment]
Personally I'm not too interested in this feature. The point of TubeSync was to be like Sonarr or Radarr, a PVR that stuffs media into Plex or Jellyfin and not play media itself. You don't watch films in Radarr for example. I can see why some people might want this feature, and it may get added to the end of the wishlist, but I wouldn't hold my breath on it appearing any time soon unless someone wants to code it as an extension and submit a decent PR for it.

**Comment by Code-Slave** [comment]
I think a good help will be embedding metadata. I just cant get jf or emby to read the nfos.

**Comment by meeb** [comment]
As soon as `youtube-dl` merges the required patches for that, it'll be added here as well.

**Comment by Code-Slave** [comment]
can you write a maintenance script that embeds into existing dl's or do i have to dl 1k vids again

**Comment by meeb** [comment]
Maybe, and a very maybe maybe at that. There is no way with `youtube-dl` to retroactively embed metadata, however, it is technically possible to embed metadata with other tools and the data is in the TubeSync database. I will look into that further, but it'll be down on the priority list for now.

**Comment by Cycor** [comment]
if you run youtube-dl again and the file exists it embeds the data without downloading the video again.
yt-dlp can use an existing json.info file but I haven't tried that yet.

**Comment by meeb** [comment]
Yeah there are open issues to explore the forks of `youtube-dl` like `yt-dlp` that do have additional features and I plan to use some of their functionality where possible.
